Wiki/Testnet.md

```diff
@@ -0,0 +1,180 @@
+    ## https://gridcoin.us/Wiki/Cofig-File
+    ##
+    ## Default testnet gridcoinresearch.conf location:
+    ##
+    ##  Windows: %AppData%\GridcoinResearch\testnet\
+    ##  Linux:   ~/.GridcoinResearch/testnet/
+    ##  macOS:   /Users/USERNAME/Library/Application/Support/GridcoinResearch/testnet/
+    ##
+    ## Single # lines are commands, remove the # in Front of the Command to use it
+    ## Double ## lines are comments
+    ##
+    #############################################################################
+    ####################### Required Settings (All OS's) ########################
+    #############################################################################
+
+    ## You must set your username to assist developers tracking down problems
+    org=CHANGE_ME
+
+    ## Decrease ban expiration time. The default 24 hours is too long for testnet.
+    ## Nodes are banned for sending bad data (seconds)
+    bantime=60
+
+    ## These Addnodes are for TESTNET only. See Slack for further details.
+    addnode=addnode-us-central.cycy.me
+    addnode=ec2-3-81-39-58.compute-1.amazonaws.com
+    addnode=gridcoin.network
+    addnode=gridhost.ddns.net
+    addnode=tarmoilves.eu
+    addnode=testnet.dihelix.com
+
+    ## Port 32748/TCP required for Inbound Connections
+    ## (Not required but highly recommended)
+    listen=1
+
+    #############################################################################
+    ######################## Optional BOINC settings ############################
+    ########### (Required if BOINC installed to non-default location) ###########
+    #############################################################################
+
+    ## Windows (Note the double backslashes are neccessary)
+    #boincdatadir=C:\\ProgramData\\BOINC\\
+    #boincappdir=C:\\Program Files\\BOINC\\
+
+    ## Linux
+    #boincdatadir=/var/lib/boinc-client/
+
+    ## macOS
+    #boincdatadir=/Library/Application Support/BOINC Data/
+
+# Difference To Mainnet
+
+Note that this is not an exhaustive list
+
+* The coins on testnet aren't worth anything like the ones on mainnet are
+    * Lets testing happen without risking anything valuable
+
+* Running on new or sometimes experimental code
+    * This can make testnet more unstable than mainnet
+    * Sometimes the wallet will receive more bad data than on mainnet 
+
+* Cooldown happens faster
+    * Makes it easier to test things relating to staking
+
+* Addresses are formatted differently
+    * Helps prevents you from accidentally sending coins to a testnet address on
+      mainnet and vice versa
+
+* Testnet doesn't have any checkpoints
+    * Helps with creating private chains for small scale testing 
+
+# Contacts
+
+General questions about testnet can often go on the `#testnet` channel on Slack.
+
+If you have specific concerns or questions, these can be directed to the testnet 
+coordinated. As of 2020, @Barton26 is the Gridcoin testnet coordinator. He helps 
```

Comment:
Note that after Fern this segment will need to be changed

Reply:
Are you stepping down from testnet coordinator, is someone else joining, or something else?

Reply:
I am stepping down from the position after the release of Fern

Reply:
Oh should I just remove the section now then or wait until after Fern?

Reply:
your call

Reply:
I think I'm just going to remove that part now and keep the general questions bit